test/distributed/_composable/fsdp/test_fully_shard_training.py

```diff
@@ -590,14 +589,18 @@ def test_train_parity_with_activation_checkpointing(self):
         """
         self.run_subtests(
             {
-                "reshard_after_forward": [True, False, 2],
+                "reshard_after_forward": [True, False],
```

Comment:
since we are removing `reshard_after_forward=2`, curious is any composability issues?

Reply:
I think since this test always runs on 2 GPUs, there is no point to test `reshard_after_forward=2`.
https://github.com/pytorch/pytorch/blob/c2378e94fe35b89bc4506fe8a8784496da3b0d31/test/distributed/_composable/fsdp/test_fully_shard_training.py#L576-L581
just removing to avoid the duplication

Reply:
got you